interface/patient_tracker/patient_tracker.php

```diff
@@ -175,252 +176,186 @@
-                                    <div class="text-left blockquote" style="width: 65%; margin: 5px auto;">
-                                        <a href="https://medexbank.com/cart/upload/index.php?route=information/campaigns&amp;g=rem"
-                                           target="_medex">
-                                            <?php echo $current_events; ?>
-                                        </a>
+                                </div>
+                                <div class="col-sm-<?php echo attr($col_width); ?>">
+                                    <div class="input-append">
+                                            <?php
+                                            if ($GLOBALS['ptkr_date_range'] == '1') {
+                                                $type = 'date';
+                                                $style = '';
+                                            } else {
+                                                $type = 'hidden';
+                                                $style = 'display:none;';
+                                            } ?>
+                                            <div class="form-group row mt-md-4" style="<?php echo $style; ?>">
+                                                <label for="flow_from" class="col-lg-2 col-sm-3 col-2 col-form-label text-sm-right mx-sm-0 px-sm-0"><?php echo xlt('From'); ?>:</label>
+                                                <div class="col-xl-3 col-lg-5 col-md-6 col-sm-7 col-4 ml-1 px-0">
+                                                    <input type="text" id="form_from_date" name="form_from_date" class="datepicker form-control form-control-sm text-center" value="<?php echo attr(oeFormatShortDate($from_date)); ?>"/>
+                                                </div>
+                                            </div>
+
+                                            <div class="form-group row" style="<?php echo $style; ?>">
+                                                <label for="flow_to" class="col-lg-2 col-sm-3 col-2 col-form-label text-sm-right mx-sm-0 px-sm-0"><?php echo xlt('To{{Range}}'); ?>:</label>
+                                                <div class="col-xl-3 col-lg-5 col-md-6 col-sm-7 col-4 ml-1 px-0">
+                                                    <input type="text" id="form_to_date" name="form_to_date" class="datepicker form-control form-control-sm text-center" value="<?php echo attr(oeFormatShortDate($to_date)); ?>"/>
+                                                </div>
+                                            </div>
+
+                                            <div class="form-group row" style="<?php echo $style; ?>">
+                                                <div class="col-sm-4 col-3 offset-lg-1 offset-md-3 offset-2">
```

Comment:
I think the messy grid layout let the jumbotron looks like it didn't center.

I adjusted the grid system. If suitable I will update the new commit.
![image](https://user-images.githubusercontent.com/22230889/85336752-c442b900-b511-11ea-8fd7-6893504c3ae9.png)

Reply:
Looks a lot better -- just needs a bit of padding on the inside and should be good